src/apiPort.js

```diff
@@ -45,17 +45,20 @@ class ApiPort {
     apis.servers[0] = { url: this.get('API_SERVER_URL') }
 
     for (const path of Object.keys(apis.paths)) {
-      let params = apis.paths[path].parameters || []
+      const params = apis.paths[path].parameters || []
```

Comment:
This is just a small refactory to simplify this for  operation it is not part of the `only` tag feature.